cargo-marker/README.md

```diff
@@ -0,0 +1,77 @@
+# Cargo Marker
+
+[![Crates.io](https://img.shields.io/crates/v/cargo-marker.svg)](https://crates.io/crates/cargo-marker)
+<!--
+FIXME(xFrednet): Add license shield, once crates.io also says:
+[![License: MIT OR Apache-2.0](https://img.shields.io/crates/l/cargo_marker.svg)](#license)
+-->
+
+*cargo_marker* is the CLI tool for [Marker], an experimental linting interface for Rust. It seamlessly integrates with the Rust compiler to provide custom linting capabilities for your Rust projects.
+
+> **Note**
+>
+> The project is in the very early stages of development, some things are still missing and the API is not stable yet.
+
+[Marker]: https://github.com/rust-marker/marker
+
+## Key Features
+
+* **Seamless Integration**: *cargo_marker* effortlessly integrates with your existing workflow to leverage Marker's linting capabilities and ensure code quality.
```

Comment:
Might be good to elaborate on this. I can see where the next two points come from by reading this text, not as much for this point.

Reply:
Yeah, this one is not that clear, especially in comparison to the others. Thank you for pointing that out :)